app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :rememberable, :validatable, :trackable
+
+  belongs_to :organisation, optional: true
```

Comment:
I've put this as optional as I don't think admins will need to belong to an organisation. There may be others who also don't belong to an organisation (viewers?)